isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncArchiveUpdated.trn

```diff
@@ -0,0 +1,107 @@
+#
+# This translation file is used to produce the ISIS 3 Archive label group
+# for the Hayabusa 2 ONC-W1, ONC-W2, and ONC-T cameras.
+#
+
+Group = ProducerId
+  Auto
```

Comment:
Same for this one, you can probably extend this to all the "updated" files

Reply:
ditto